catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js

```diff
@@ -36,6 +36,8 @@ var mtgeo = require('mt-geo');
 var defaultColor = '#3c6dd5';
 var eyeOffset = new Cesium.Cartesian3(0, 0, 0);
 var pixelOffset = new Cesium.Cartesian2(0.0, 0);
+var viewer;
```

Comment:
❗️ We can't pull these variables (`viewer`, `currentMapCenter`) out as singletons, otherwise they'll be shared by all the open cesium maps.

Reply:
Added openlayers support and changed a few things around here: 6d7b17e72b9753c6c118cebdf28b837ede316650